spec/lib/msf/core/exploit/remote/remote/http/http_cookie_jar_spec.rb

```diff
@@ -0,0 +1,262 @@
+require 'spec_helper'
+require 'msf/core/exploit/remote/http/http_cookie_jar'
+require 'msf/core/exploit/remote/http/http_cookie'
+require 'faker'
+
+def random_string(min_len=1, max_len=12)
+  (('a'..'z').to_a + ('A'..'Z').to_a).shuffle[min_len,max_len].join
+end
```

Comment:
Looks like this introduced non-deterministic tests that can't be debugged with `rspec --bisect` 👀